data/images/2024-uswds-monthly-call-aug-title-card.yml

```diff
@@ -0,0 +1,12 @@
+
+  # https://s3.amazonaws.com/digitalgov/2024-uswds-monthly-call-aug-title-card.png
+  # Image shortcode: {{< img src=2024-uswds-monthly-call-aug-title-card >}}'
+  date     :  2024-08-12 15:37:52 -0400
+  uid      :  2024-uswds-monthly-call-aug-title-card
+  width    :  1200
+  height   :  628
+  format   :  png
+  credit   :  
+  caption  :  
+  alt      :  
```

Comment:
**question (non-blocking):** Do we need alt text here?

The card for June included one:
https://github.com/GSA/digitalgov.gov/blob/018c4b5e0aa8977a7ebcd14a4b0d0255d5a7d1d9/data/images/2024-uswds-monthly-call-june-title-card.yml#L11

Reply:
Added in [c8b1157](https://github.com/GSA/digitalgov.gov/pull/7856/commits/c8b11579be8f2cd5d3be423ca715b041cba916c4).